hook!-full-game/src/Player/Skills/Base.gd

```diff
@@ -72,7 +72,7 @@ func _unhandled_input(event: InputEvent) -> void:
 
 
 func _physics_process(delta):
-	var move_direction: Vector2 = player.get_move_direction()
+	var move_direction: Vector2 = player.get_move_direction() if not player.dead else Vector2.ZERO
```

Comment:
Your state of the project is outdated, you should always rebase on the latest master before working more on a PR. The skill system doesn't exist anymore since 1+ week ago